README.md

```diff
@@ -1 +1,60 @@
-# Hello_Microverse
\ No newline at end of file
+Hello_Microverse
+
+_______________________________________________________________________________
+
+
+Additional description about the project and its features.
+
+Built With
+Major languages
+Frameworks
+Technologies used
+
+_______________________________________________________________________________
+Live Demo
+Live Demo Link
+_______________________________________________________________________________
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+
+Setup
+
+Install
+
+Usage
+
+Deployment
+
+
+_______________________________________________________________________________
+👤 Author1
+
+GitHub: @Rizwanhasan.learn  
+
+Twitter: @rizwanhasan21
+
+LinkedIn: Rizwan Hasan
```

Comment:
- [ ]  As I have mentioned, please properly format headings and add links to your pages. Here how you can add links and format Authors section: 

```
## Authors

👤 **Rizwan Hasan**

- GitHub: [@Rizwan-learn](https://github.com/Rizwan-learn)
- LinkedIn: [LinkedIn](PASTE YOUR  LinkedIn LINK HERE)
```

Reply:
ReadMe has been updated